pages/docs/user-docs/docs-create-an-image.md

```diff
@@ -5,15 +5,16 @@ permalink: create-image
 folder: docs
 ---
 
-Singularity images are single files which physically contain the container. Unlike Docker that puts images together from layers, abstractly shown on your computer with `docker -ps`, a Singularity image is just a file that can be sitting on your Desktop, in a folder on your cluster, or elsewhere.
+A Singularity image is a single file that contains a virtual file system. It is the basis for a Singularity container. After creating an image, you can install an operating system, applications, and save meta-data into it.   
```

Comment:
This one bit sounds strange to me, because the file is the image is the container. What about something like:

>>> A Singularity image, which can be referred to as a "container," is a single file that contains a virtual file system. After creating an image you can install an operating system, applications, and save meta-data with it.